Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde_derive = "^1.0.75"
 
 [dev-dependencies]
 ron = "^0.4"
-reqwest = "^0.8"
+reqwest = { version = "^0.10", features = ["blocking"] }
```

Comment:
What is the reason for updating to newer version of reqwest?

Reply:
As I said, it wasn't compiling for me on latest Arch Linux. Here is the error:
```
error: failed to run custom build command for `openssl v0.9.24`

Caused by:
  process didn't exit successfully: `/mnt/hdd1/Documents/rust/reqwest_test/target/release/build/openssl-d662e4b31bdd8b2a/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to detect OpenSSL version', /home/friz64/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.24/build.rs:16:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

warning: build failed, waiting for other jobs to finish...
error: build failed
```
[This is the fault of the old reqwest version.](https://github.com/sfackler/rust-openssl/issues/994#issuecomment-429634655) Also switching to `rustls-tls` in `reqwest` 0.9+ doesn't make it compile either.

I don't understand why a CI with a 2 year old Rust version is necessary.

Reply:
I admit, I somehow missed your mention of build errors on Arch Linux. Sorry, I guess I was a bit tired.

> I don't understand why a CI with a 2 year old Rust version is necessary.

That's the version that was current (give or take) at the time of first release and I saw no reason to change this minimum since then. Update of a dev dependency just so tests build and run on an OS neither I nor CI uses, doesn't sound like particularly good reason. Admittedly, the only public user of the crate I know of is `ash` and that's likely on a newer version of Rust, so it probably wouldn't cause problems to anyone. I wouldn't be opposed to bumping the minimum version if there was an actual reason in code. I would like to see `reqwest` gone from dependencies, rather than updated, but I have yet to see any alternative I would be happy with (given my very limited needs).

Also, I don't like having multiple things mixed in the same PR. `reqwest` update is not related to adding support for new attribute and it made me think about accepting the PR way more than it otherwise would have.

Reply:
> Update of a dev dependency just so tests build and run on an OS neither I nor CI uses, doesn't sound like particularly good reason.

Well, the tests won't work on any Linux based OS that runs OpenSSL 1.1.1, which came out in September 2018. This results in pretty much any contributor running Linux not being able to run the tests. They work in the CI because it runs Ubuntu 16.04, which is 4 years old by now.

> I would like to see `reqwest` gone from dependencies, rather than updated, but I have yet to see any alternative I would be happy with (given my very limited needs).

If we were to still support Rust 1.25 we would need to [find a library](https://lib.rs/web-programming/http-client) that
- still compiles and works on Rust 1.25
- supports TLS (https)
- is easy to use / suits our needs of sending a http get request

Still, an updated Rust is just one `rustup update` away. I don't think it ever makes sense to support old rust versions.

Reply:
I went on and had a look the crate list you linked and found one matching almost all my requirements. It did require slight increase in minimum supported rustc version, but given that it removes vast majority of dependencies, I think it's well worth it. Given that I plan to make a few more breaking changes, I figured I might as well do it now.

If you'd like to test whether it compiles fine on newer distros, here's the commit: https://github.com/krolli/vk-parse/commit/adeb9aa3af2c76332cc48ab0337d520906197bc4.

Reply:
I can confirm that the tests compile and pass! Thank you for your work.